test/cri-containerd/logging_binary_test.go

```diff
@@ -0,0 +1,144 @@
+// +build functional
+
+package cri_containerd
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	runtime "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+)
+
+// This test requires compiling a helper logging binary which can be found
+// at test/cri-containerd/helpers/log.go. Copy log.exe as "sample-logging-driver.exe"
+// to ContainerPlat install directory or set "TEST_BINARY_ROOT" environment variable,
+// which this test will use to construct logPath for CreateContainerRequest and as
+// the location of stdout artifacts created by the binary
+func Test_Run_Container_With_Binary_Logger(t *testing.T) {
+	client := newTestRuntimeClient(t)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	logBinaryRoot := os.Getenv("TEST_BINARY_ROOT")
+	if logBinaryRoot == "" {
+		logBinaryRoot = "/ContainerPlat"
+	}
+
+	binaryPath := logBinaryRoot + "/sample-logging-driver.exe"
+
+	if _, err := os.Stat(binaryPath); os.IsNotExist(err) {
+		t.Skip("skipping: sample logging driver missing")
+	}
+
+	logPath := "binary://" + binaryPath
+
+	type config struct {
+		TestConfig
```

Comment:
I think we decided to axe the global test config but removing this definitely doesn't have to be done in this PR